x-pack/legacy/plugins/transform/public/app/lib/kibana/kibana_provider.tsx

```diff
@@ -61,7 +58,7 @@ export const KibanaProvider: FC<Props> = ({ savedObjectId, children }) => {
     const kibanaContext: InitializedKibanaContextValue = {
       indexPatterns,
       initialized: true,
-      kbnBaseUrl: npStart.core.injectedMetadata.getBasePath(),
+      kbnBaseUrl: appDeps.core.injectedMetadata.getBasePath(),
```

Comment:
`basePath` can be got from `http`, i believe `injectedMetadata` is deprecated.

Reply:
Fixed in 5757826.